tools/fleetctl-docker/README.md

```diff
@@ -0,0 +1,26 @@
+## fleetdm/fleetctl
+
+This docker image allows to run `fleetctl` in a Linux environment that has all
+the necessary dependencies to package `msi`, `pkg`, `deb` and `rpm` packages.
+
+### Usage
+
+```
+docker run fleetdm/fleetctl command [flags]
+```
+
+Build artifacts are generated at `/build`. To get a package using this image:
+
+```
+docker run -v "$(pwd):/build" fleetdm/fleetctl package --type=msi
+```
+
+### Building
+
+This image needs to be built from the root of the repo in order for the build
```

Comment:
alternatively, we could put this Dockerfile along the others at the root of the repo and use `make docker-build-release` and `docker-push-release`. Happy to change accordingly if we think is worth it.

Reply:
Looks good as is.